src/BenchmarksDriver/Program.cs

```diff
@@ -747,6 +749,14 @@ public static int Main(string[] args)
                         Log("WARNING: '--self-contained' has been set implicitly as the mono runtime is used.");
                         Console.ResetColor();
                     }
+                    if (monoOption_LlvmJit.HasValue())
```

Comment:
Just add an `||` with the previous statement. And if you merge both arguments then that works too.